NEWS.md

```diff
@@ -248,6 +253,11 @@ Additional options include:
 
     ([#4460](https://github.com/syslog-ng/syslog-ng/pull/4460))
 
+  * `opensearch`: Added a new destination.
```

Comment:
This makes it 4.4.0 in terms of semantic versioning. But I understand if we want to keep 4.3.1.

Reply:
We'll go with 4.3.1, and repeat this news entry in 4.4.0.